### DISCUSSION TITLE
Speak Reading glycemia value and variation values

### DISCUSSION THREAD

**Discussion by henriquejourdan** [NightscoutFoundation/xDrip#2333] [Ideas]
Is it possible to speak/read glycemia value and variation values (sequencial)?
For example: "120" + "rising or up or plus" + "4"
THANKS

**Comment by Navid200** [comment]
Have you enabled the following setting?

Settings &#8722;> Speak Readings &#8722;> Speak trend arrow name

**Comment by henriquejourdan** [comment]
>Navid2000. Yes I tried, but it reads rissing fast, stable, falling...... NOT the variation value (good for predictions).
Thanks

**Reply by Navid200** [reply]
I see what you mean.  Thanks

**Reply by Navid200** [reply]
@henriquejourdan Would you please open an issue, as a feature request?  https://github.com/NightscoutFoundation/xDrip/issues
Please include in it a link to this thread.

Thanks
